pkg/util/validation/limits.go

```diff
@@ -33,6 +33,13 @@ type Limits struct {
 
 // RegisterFlags adds the flags required to config this to the given FlagSet
 func (l *Limits) RegisterFlags(f *flag.FlagSet) {
+	// lite uses both ingester.Config and distributor.Config.
+	// Both of them has Limits, so calling RegisterFlags on them triggers panic.
+	// This check ignores second call to RegisterFlags on IngesterClientConfig and then populates it manually with SetClientConfig
```

Comment:
Need to update `IngesterClientConfig` and `SetClientConfig` in this line. Also in the other comment since the method name changed.